README-ckad.md

```diff
@@ -0,0 +1,105 @@
+
+# Certified Kubernetes Application Developer (CKAD)
+A  curated list of resources that will help you to face your Kubernetes certification. Based in the newest policy about 'Resources Allowed', current list will give preference
+to those materials coming from the allowed sources [kubernetes.io](kubernetes.io). Others like Youtube videos, blog resources could be referenced in expectional case due to I still found them useful in my k8s learning journey.
+
+
+_Before to start_ 
+
+Ensure you have the right version of Kubernetes documentation selected (e.g. v1.9 for the current exam) especially for API objects and annotations.
+
+
+## Exam Objectives
+
+These are the exam objectives you review and understand in order to pass the test. The objectives are current as of December 12, 2017.
```

Comment:
```
Following are the exam objectives you can review and understand in order to pass the test. These objectives are based on December 12, 2017 date.
```

Reply:
cool, I am travelling to Dubai AWS summit, will review it when I am back.  did you try the CKAD exam?